README.md

```diff
@@ -97,6 +100,7 @@ The configuration defines the various components of the project you want to mana
 | healthChecks          | The various checks to run to ensure the container is healthy (see below for list) |             |
 | terminationSignals    | Signals which should be sent to a specific container when it should be shut down  |             |
 | terminationChecks     | The various checks to run to ensure that the container is ready to be shut down   | connections |
```

Comment:
Do you envision volume containers needing the ability to be rolling upgraded in some fashion? If not, it should be fine having them refer to the Docker container names directly.